src/arrays/programs_involving_data_sequences/hard/q7/SquareUp.java

```diff
@@ -0,0 +1,37 @@
+package arrays.programs_involving_data_sequences.hard.q7;
+
+/*
+Write a Java program that calls a method called doubler that takes as input an integer array, and squares each element
```

Comment:
Change question text to:
"Write a Java program that asks the user to input 10 integers and stores them in an array. Then it should pass them to a method called doubler takes an input of an integer array and stores the square of each element in a new array which it then returns. After calling the method, the main method should print the contents of the new array separated by a space. Your method should be general enough to work with any size of array."